whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/query/AbuseQuerySpec.groovy

```diff
@@ -907,4 +907,30 @@ class AbuseQuerySpec extends BaseQueryUpdateSpec {
         queryObject("--abuse-contact AS200", "abuse-mailbox", "more_abuse@test.net")
     }
 
+    def "inverse query for aut-num using person"() {
```

Comment:
Can you also add a test case for the usual reference from an organisation to a role using an abuse-c reference?